cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -770,11 +793,12 @@ def test__validate_tissue_position_required(self, tissue_position_name):
             f"obs['{tissue_position_name}'] {ERROR_SUFFIX_VISIUM_AND_IS_SINGLE_TRUE_REQUIRED}." in validator.errors[0]
         )
 
-    def test__validate_tissue_position_not_required(self):
```

Comment:
Moved to parameterized to avoid `["EFO:0010961", "EFO:0030062"]` triggering unique error.